docs/APIs/tasks/Timeout.md

```diff
@@ -1,8 +1,8 @@
-# TimeOut
+# Timeout
 
 Use the Timeout interface to set up an interrupt to call a function after a specified delay.
 
-Any number of Timeout objects can be created, allowing multiple outstanding interrupts at the same time.
+You can create any number of Timeout objects, allowing multiple outstanding interrupts at the same time.
```

Comment:
Which would imply that this should also be TimeOut. I may be wrong about the name.